src/routes/loginRouter.ts

```diff
@@ -0,0 +1,29 @@
+
+import bcrypt from 'bcrypt';
+import { PrismaClient } from '@prisma/client';
+import express, { Request, Response } from 'express';
+import { checkUser, checkPassword, generateToken } from '../services/loginServices';
+
+const router = express.Router();
+
+router.post('/login', async (req: Request, res: Response) => {
+  const prismaClient = new PrismaClient();
+  const { email, password } = req.body;
+
+  const user = await checkUser(email, prismaClient);
+
+  if (!user) {
+    return res.status(400).send('User not found');
```

Comment:
```suggestion
    return res.status(404).send('User not found');
```
Check the error codes in the comment above and update the controllers, please.